src/config/dynamic-theme-fixes.config

```diff
@@ -1764,6 +1764,18 @@ INVERT
 
 ================================
 
+decathlon.in
+
+CSS
+.media, .swiper-slide {
```

Comment:
Replace this with:
```CSS
.common_card .card_img img {
    mix-blend-mode: none !important;
}
```

Reply:
I tried it, but it doesn't work :(. The images on the search results and the "Advantages" section are as they were before the fix. I also tried setting the mix-blend-mode property for the classes img-fluid, media and media-left, but to no avail

Reply:
What about:
```CSS
img {
    mix-blend-mode: normal !important;
}
```

Reply:
This does the job.
The images in the "advantages" section [here](https://www.decathlon.in/p/8325111/bikes-brakes-and-brake-pads/disc-brake-pads-compatible-with-shimano-deore-tektro) do not blend in quite as well as they would without dark mode -- is this a concern? This is only an aesthetic problem, and not relating to usability or anything of the sort.

Reply:
[I've committed the change](https://github.com/Vedanth-P/darkreader/commit/e10a78e52064bf6324f286cc408a32bbddbb4d87) to Vedanth-P/darkreader, branch patch-3 -- the same branch that was created when I submitted the pull request. Is that what I'm supposed to do (I'm kinda new to GitHub and Git)?

Reply:
> [I've committed the change](https://github.com/Vedanth-P/darkreader/commit/e10a78e52064bf6324f286cc408a32bbddbb4d87) to Vedanth-P/darkreader, branch patch-3 -- the same branch that was created when I submitted the pull request. Is that what I'm supposed to do (I'm kinda new to GitHub and Git)?

Correcto.